src/core/memory.d

```diff
@@ -895,6 +895,8 @@ void pureFree(void* ptr) @system pure @nogc nothrow
     // See also: https://issues.dlang.org/show_bug.cgi?id=17956
     void* z = pureMalloc(size_t.max & ~255); // won't affect `errno`
     assert(errno == fakePureErrno()); // errno shouldn't change
+
+    version (DragonFlyBSD) {} else    // FIXME : z seems to be allocated and not null. Why would size_t.max - 2 not be allowed to allocate ? or is the assertion wrong
```

Comment:
FIXME : 
z seems to be allocated and not null.
Why would size_t.max - 2 not be allowed to allocate ? or is the assertion wrong